catroid/src/main/java/org/catrobat/catroid/ui/controller/LookController.java

```diff
@@ -706,24 +682,11 @@ private File copyLookBackPack(LookData selectedlookData, String newLookDataName,
 		return null;
 	}
 
-	public void copyLook(int position, List<LookData> lookDataList, final Activity activity, LookFragment fragment) {
-		LookData lookData = lookDataList.get(position);
+	public LookData copyLook(LookData look, String copyNameAddition) throws IOException {
+		String copiedLookName = look.getLookName() + "_" + copyNameAddition;
```

Comment:
you could use getUniqueLookName(String name, List<LookData> scope) from the backpack branch.